Source/WebCore/rendering/ReferencedSVGResources.h

```diff
@@ -71,10 +74,12 @@ class ReferencedSVGResources {
     static RefPtr<SVGClipPathElement> referencedClipPathElement(TreeScope&, const ReferencePathOperation&);
     static RefPtr<SVGMarkerElement> referencedMarkerElement(TreeScope&, const String&);
     static RefPtr<SVGMaskElement> referencedMaskElement(TreeScope&, const StyleImage&);
+    static RefPtr<SVGElement> referencedPaintServerElement(TreeScope&, const String&);
 #endif
 
 private:
     static RefPtr<SVGElement> elementForResourceID(TreeScope&, const AtomString& resourceID, const QualifiedName& tagName);
+    static RefPtr<SVGElement> elementForResourceIDs(TreeScope&, const AtomString& resourceID, const QualifiedNames& tagNames);
```

Comment:
I wonder if we can just always use the QualifiedNames variant?

Reply:
Yeah, but the former variant saves the QualifiedNames traversal

Reply:
We could, but the former saves an unnecessary Vector<QName> creation + traversal?